src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -51,8 +56,23 @@ StartRemoteTransactionBegin(struct MultiConnection *connection)
 	 * side might have been changed, and that would cause problematic
 	 * behaviour.
 	 */
-	if (!SendRemoteCommand(connection,
-						   "BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED"))
+	appendStringInfo(beginAndSetDistributedTransactionId, "%s;",
+					 "BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED");
```

Comment:
Don't quite understand why you're using %s here at all, I'd just go for `appendStringInfoString()`.